tools/github/repositories.txt

```diff
@@ -0,0 +1,30 @@
+Azure/azure-sdk-for-net
```

Comment:
The fact this file here means:

1. Do use `$PSScriptRoot/repositories.txt` and change my suggested text above, and
2. What was the point of adding complexity to the script to read from a file as opposed to just adding them to the default `$Repositories` parameter? You listed all the private repos here anyway. I don't see value in this over just changing the default parameter value. Unless you wanted to share the list, but there's better ways to do that such that all the parameters for the scripts could share.

Reply:
The file is the "source of truth" for repositories that are centrally managed.  I've been using it to guide automation using one source across scripts.   For most scenarios, that's the default set.   For others - like the issues/projects - we probably don't want all the weird and private ones.

If you've got thoughts on refactoring for a different method of sharing, I'm all for it.   I'm going to leave this here for now and we can iterate.

Reply:
If this is easier for central management, just keep it as is but make the adjustments to the `$RepositoryFile` parameter in both scripts like I mentioned above.

Reply:
Very late to the conversation, but @jsquire do you have concerns on adding Cadl repository to this list?

Reply:
@maririos:  I have no objections, so long as we want to start to manage the Cadl repository centrally.   That will include labels, bot rules, and all the trimmings eventually.

Reply:
In the meantime, just run the script - whether you change respositories.txt or not - passing just `-Repositories 'Azure/azure-cadl' (or whatever it is) to the script.

Reply:
FYI @ckairen for you to consider as part of your Cadl integration work.